### DISCUSSION TITLE
i cant find my pages

### DISCUSSION THREAD

**Discussion by Mohamed-gp** [community/community#60942] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I can find my web in my repos files but I couldn't find it in other accounts like my friend's PC or Incognito mode these photos to describe the problem am sorry if it's not the place to report a problem
<img width="936" alt="Screenshot 2023-07-15 202338" src="https://github.com/community/community/assets/121584128/a9fbfaba-6079-42ca-b158-eb877cc01e62">
<img width="903" alt="Screenshot 2023-07-15 202516" src="https://github.com/community/community/assets/121584128/06bbf4d1-ad2b-4c32-abde-7f3f371e0932">

**Comment by radar-blip** [comment]
Its working as it should for me. 
https://mohamed-gp.github.io/Elzero_Template_Three/

**Reply by Mohamed-gp** [reply]
thnx bro I think the problems because of incongnito mode
